pkg/network/nmstate.go

```diff
@@ -74,13 +104,33 @@ func removeDaemonSetHandlerWorker(ctx context.Context, client k8sclient.Client)
 	return []error{}
 }
 
-func cleanUpNMState(conf *cnao.NetworkAddonsConfigSpec, ctx context.Context, client k8sclient.Client) []error {
+func removeStandaloneHandler(ctx context.Context, client k8sclient.Client) []error {
+	namespace := os.Getenv("OPERAND_NAMESPACE")
+	name := "nmstate-handler"
+	kind := "DaemonSet"
+
+	return removeAppsV1Resource(ctx, client, name, namespace, kind)
+}
+
+func removeStandaloneWebhook(ctx context.Context, client k8sclient.Client) []error {
+	namespace := os.Getenv("OPERAND_NAMESPACE")
+	name := "nmstate-webhook"
+	kind := "Deployment"
+
+	return removeAppsV1Resource(ctx, client, name, namespace, kind)
+}
+
+func cleanUpNMState(conf *cnao.NetworkAddonsConfigSpec, ctx context.Context, client k8sclient.Client, clusterInfo *ClusterInfo) []error {
 	if conf.NMState == nil {
 		return []error{}
 	}
 
 	errList := []error{}
 	errList = append(errList, removeDaemonSetHandlerWorker(ctx, client)...)
+	if clusterInfo.NmstateOperator {
+		errList = append(errList, removeStandaloneHandler(ctx, client)...)
+		errList = append(errList, removeStandaloneWebhook(ctx, client)...)
+	}
```

Comment:
We have bump CNAO to latest nmstate and now there is a new Deployment "cert-manager" it has to be added here.

Reply:
Done.